docs/source/logos.rst

```diff
@@ -3,29 +3,35 @@
 Images and Logos
 ================
 
+Here are some commonly used Dask icons and logos
+(see the `style guide <dask.org/style-guide>`).
```

Comment:
I think we need a trailing `_` for this link to render properly 

```suggestion
(see the `Dask style guide <dask.org/style-guide>`_ for more details).
```

Reply:
:) thanks James!